components/dada-ir/src/code/validated/op.rs

```diff
@@ -37,6 +37,8 @@ impl std::fmt::Display for Op {
 
 define_operators! {
     EqualEqual => "==",
+    GreaterEqual => ">=",
+    LessEqual => "<=",
```

Comment:
I am reminded that it would be really cool if the Python thing of `3 <= X <= 5` worked-- but that's independent from this PR.